### ISSUE TITLE
Error in Server Console: java.lang.NullPointerException: null

### ISSUE THREAD

**Issue by Wissi** [sokratis12GR/ArmorPlus#221]
**Affected Versions (Do not use "latest"):**:
  - **ArmorPlus**: armorplus-1.12.2-11.22.0.58
  - **TheDragonLib**: thedragonlib-1.12.2-5.3.0
  - **Minecraft**: 1.12.2.
  - **Forge**: forge-1.12.2-14.23.5.2836
  - (Optional) Just Enough Items (JEI): jei_1.12.2-4.15.0.278
  - (Optional) Tinkers' Construct: TConstruct-1.12.2-2.12.0.135
  - (Optional) CraftTweaker (CrT): CraftTweaker2-1.12-4.1.17

**Affected Side (Client or Server)**:
- [ ] Client
- [X] Server

**What happens**:
Armorplus produces this error and lags the game. Console error that makes the server lag when it's produced. 

**What you expected to happen:**:
No error?

**If its a crash, post your crash report (via. pastebin/gist/etc)**:
[latest.log.txt](https://github.com/sokratis12GR/ArmorPlus/files/3160708/latest.log.txt)

**Your configuration files from `config/armorplus/` (`registry.cfg` mainly)**:
[registry.cfg.txt](https://github.com/sokratis12GR/ArmorPlus/files/3160700/registry.cfg.txt) Added the .txt extension so it could upload.

**(Optional) Mod List**:

**Comment by sokratis12GR** [comment]
Working on a fix for this issue.

**Comment by sokratis12GR** [comment]
Question. How to exactly reproduce this issue?
After the player dies?
After a server restart?
After a player is re-joining the server world?

**Comment by Wissi** [comment]
I checked the console and most times it's when a player dies and other times when a player joins.
I went ahead and checked all configs and `/config/armorplus/entities.cfg` had:
```
mob_drops {

    trophy {
        # Enable/Disable the trophy dropped when killing a boss (vanilla bosses & some of the wip armorplus bosses)
        B:enableTrophyDrops=false
```
Changed that to True and it seems ERROR msg in console has dissapeared.  (false by default and it's when error spams console).

**Comment by sokratis12GR** [comment]
🤦‍♂ I thought trophies were enabled by default, oh well now that is a big bug...
Will be fixed next update.

**Comment by sokratis12GR** [comment]
Should be fixed: [ArmorPlus 1.12.2-11.23.0.59](https://minecraft.curseforge.com/projects/armorplus/files/2714162)
Let me know if it was fixed or not

**Comment by Wissi** [comment]
Awesome work and mod.
Yes, it is fixed now. 
Thank you very much for a quick response and fix.
